e2e/app/component/admin-table.ts

```diff
@@ -15,83 +15,44 @@ export default class adminTable extends Table {
     return this.userLinkElement.asElement();
   }
 
-  getFrozenHeader(): Table {
+  getHeaderRow(): Table {
     return new Table(
       this.page,
       `${this.getXpath()}
-      //div[@class="p-datatable-scrollable-view p-datatable-frozen-view"]
-      //table[@class="p-datatable-scrollable-header-table"]`
+      //div[@class="p-datatable-wrapper"]
+      //table[@class="p-datatable-table"]
+      //thead//tr`
     );
   }
 
-  getFrozenBody(): Table {
+  getTable(): Table {
```

Comment:
I believe that the old version of PrimeReact had tables for the header, body, and footer. With the new version, it uses one table and makes use of thead, tbody, and tfoot.